diff --git a/pyEcoreCapella/information/communication/communication.py b/pyEcoreCapella/information/communication/communication.py
index f6101ca..582c44f 100644
--- a/pyEcoreCapella/information/communication/communication.py
+++ b/pyEcoreCapella/information/communication/communication.py
@@ -6,8 +6,12 @@ from pyecore.ecore import *
 from behavior import AbstractSignal
 from capellacore import CapellaElement, Classifier, Relationship, Structure, VisibilityKind
 from information.datavalue import DataValueContainer
-from information import AbstractInstance
-
+from emde import Element, ExtensibleElement
+try:
+   from information import AbstractInstance
+   print('communication.{__name__} sucessfully imported AbstractInstance')
+except ImportError:
+   print('communication.{__name__} failed to import AbstractInstance')
 
 name = 'communication'
 nsURI = 'http://www.polarsys.org/capella/core/information/communication/1.4.0'
@@ -218,7 +222,7 @@ class Message(CommunicationItem):
         super().__init__(**kwargs)
 
 
-class SignalInstance(AbstractInstance):
+class SignalInstance(EObject, metaclass=MetaEClass): #CGA AbstractInstance
 
     def __init__(self, **kwargs):
 
