diff --git a/pyEcoreCapella/__init__.py b/pyEcoreCapella/__init__.py
index 78843c8..abfbcba 100644
--- a/pyEcoreCapella/__init__.py
+++ b/pyEcoreCapella/__init__.py
@@ -357,7 +357,7 @@ InterfaceAllocation._allocatedInterface.eType = Interface
 InterfaceAllocation._allocatedInterface.eOpposite = Interface.provisioningInterfaceAllocations
 InterfaceAllocation._allocatingInterfaceAllocator.eType = InterfaceAllocator
 InterfaceAllocator.provisionedInterfaceAllocations.eType = InterfaceAllocation
-InterfaceAllocator.provisionedInterfaceAllocations.eOpposite = InterfaceAllocation.allocatingInterfaceAllocator
+#InterfaceAllocator.provisionedInterfaceAllocations.eOpposite = InterfaceAllocation.allocatingInterfaceAllocator
 AbstractPhysicalArtifact.allocatorConfigurationItems.eType = ConfigurationItem
 PhysicalPort.allocatedComponentPorts.eType = ComponentPort
 
@@ -580,18 +580,18 @@ FunctionalExchange.involvingFunctionalChains.eOpposite = FunctionalChain.involve
 FunctionalExchange.allocatingComponentExchanges.eType = ComponentExchange
 FunctionalExchange.incomingComponentExchangeFunctionalExchangeRealizations.eType = ComponentExchangeFunctionalExchangeAllocation
 FunctionalExchange.incomingFunctionalExchangeRealizations.eType = FunctionalExchangeRealization
-FunctionalExchange.incomingFunctionalExchangeRealizations.eOpposite = FunctionalExchangeRealization.realizedFunctionalExchange
+#FunctionalExchange.incomingFunctionalExchangeRealizations.eOpposite = FunctionalExchangeRealization.realizedFunctionalExchange
 FunctionalExchange.outgoingFunctionalExchangeRealizations.eType = FunctionalExchangeRealization
-FunctionalExchange.outgoingFunctionalExchangeRealizations.eOpposite = FunctionalExchangeRealization.realizingFunctionalExchange
+#FunctionalExchange.outgoingFunctionalExchangeRealizations.eOpposite = FunctionalExchangeRealization.realizingFunctionalExchange
 FunctionalExchange.realizedFunctionalExchanges.eType = FunctionalExchange
 FunctionalExchange.realizingFunctionalExchanges.eType = FunctionalExchange
 FunctionalExchange.realizingFunctionalExchanges.eOpposite = FunctionalExchange.realizedFunctionalExchanges
 AbstractFunction.outFunctionRealizations.eType = FunctionRealization
-AbstractFunction.outFunctionRealizations.eOpposite = FunctionRealization.allocatingFunction
+#AbstractFunction.outFunctionRealizations.eOpposite = FunctionRealization.allocatingFunction
 AbstractFunction.inFunctionRealizations.eType = FunctionRealization
-AbstractFunction.inFunctionRealizations.eOpposite = FunctionRealization.allocatedFunction
+#AbstractFunction.inFunctionRealizations.eOpposite = FunctionRealization.allocatedFunction
 AbstractFunction.componentFunctionalAllocations.eType = ComponentFunctionalAllocation
-AbstractFunction.componentFunctionalAllocations.eOpposite = ComponentFunctionalAllocation.function
+#AbstractFunction.componentFunctionalAllocations.eOpposite = ComponentFunctionalAllocation.function
 AbstractFunction.allocationBlocks.eType = AbstractFunctionalBlock
 AbstractFunction.allocationBlocks.eOpposite = AbstractFunctionalBlock.allocatedFunctions
 AbstractFunction.involvingFunctionalChains.eType = FunctionalChain
@@ -891,9 +891,9 @@ AbstractEnd._covered.eOpposite = InstanceRole.abstractEnds
 AbstractCapabilityRealization._realizedCapability.eType = AbstractCapability
 AbstractCapabilityRealization._realizingCapability.eType = AbstractCapability
 AbstractCapability.incomingCapabilityAllocation.eType = AbstractCapabilityRealization
-AbstractCapability.incomingCapabilityAllocation.eOpposite = AbstractCapabilityRealization.realizedCapability
+#AbstractCapability.incomingCapabilityAllocation.eOpposite = AbstractCapabilityRealization.realizedCapability
 AbstractCapability.outgoingCapabilityAllocation.eType = AbstractCapabilityRealization
-AbstractCapability.outgoingCapabilityAllocation.eOpposite = AbstractCapabilityRealization.realizingCapability
+#AbstractCapability.outgoingCapabilityAllocation.eOpposite = AbstractCapabilityRealization.realizingCapability
 AbstractCapabilityExtend.extensionLocation.eType = AbstractCapabilityExtensionPoint
 AbstractCapabilityExtensionPoint.extendLinks.eType = AbstractCapabilityExtend
 AbstractCapabilityExtensionPoint.extendLinks.eOpposite = AbstractCapabilityExtend.extensionLocation
@@ -1060,12 +1060,12 @@ ActivityAllocation._activity.eType = OperationalActivity
 ActivityAllocation._activity.eOpposite = OperationalActivity.activityAllocations
 Role.roleAllocations.eType = RoleAllocation
 Role.activityAllocations.eType = ActivityAllocation
-Role.activityAllocations.eOpposite = ActivityAllocation.role
+#Role.activityAllocations.eOpposite = ActivityAllocation.role
 RoleAllocation._role.eType = Role
 RoleAllocation._role.eOpposite = Role.roleAllocations
 RoleAllocation._entity.eType = Entity
 Entity.roleAllocations.eType = RoleAllocation
-Entity.roleAllocations.eOpposite = RoleAllocation.entity
+#Entity.roleAllocations.eOpposite = RoleAllocation.entity
 Entity.allocatedOperationalActivities.eType = OperationalActivity
 Entity.allocatedOperationalActivities.eOpposite = OperationalActivity.allocatorEntities
 
